fields/field.entry_relationship.php

```diff
@@ -1294,7 +1294,13 @@ public function displayPublishPanel(XMLElement &$wrapper, $data = null, $flagWit
 			}
 			
 			$sectionsId = array_map(array('General', 'intval'), $sectionsId);
-			$sections = SectionManager::fetch($sectionsId);
+			$sections = $this->sectionManager->fetch($sectionsId);
```

Comment:
I think you can acheve a simpler solution using the SectionManager::fetch arguments. not sure tho. @nitriques wdyt?

Reply:
It should work with the sectionManager::fetch but, for an unknown reason, in the fetch method, you can't have a WHERE statement and a ORDER BY statement at the same time. Seems like a weird feature but since it is in the core of symphony, we decided to do the sort directly in the extension